docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,27 @@ The ACS will be used for V&V testing. It is defined as:
 - **Not** longitudinal (independent samples) 
 - Includes mail/online, telephone, and personal visits 
 
+**Survey - Current Population Survey (CPS)** 
+
+The CPS is a survey run by the Census Bureau and gathers data about the 
+labor force, employment and unemployment, demographics, earnings, and 
+more information. It is an important survey and therefore is being 
+added here. 
+
+To create this survey: 
+
+- Sample rate of (60,000 / 350 million)% 
+- Sample will be stratified by state to ensure an even distribution 
+- Sample at each time step (approximates monthly) for 12 time steps 
+- **Not** longitudinal (independent samples) 
+- This survey utilizes personal visits and phone calls. As this does not fit into the framework above, we will use the values for mail/online, telephone, and personal visits and then apply an overall non-response rate of 27.6%. This additional risk of non-response will be added to all simulants regardless of race/ethnicity, age, or sex 
```

Comment:
could you explain what "use the values for mail/online, telephone, and personal visits" means? i.e., where do those values come from, and how are we using them? does that mean we will sample people at rate X, where X is the rate of any response, regardless of if it was mail/online, telephone, personal visit?